src/OpenTelemetry/Logs/LogRecord.cs

```diff
@@ -28,6 +29,16 @@ public sealed class LogRecord
         internal LogRecord(DateTime timestamp, string categoryName, LogLevel logLevel, EventId eventId, object state, Exception exception)
         {
             this.Timestamp = timestamp;
+
+            var activity = Activity.Current;
```

Comment:
should we check if this information is available in Scope (starting .net 5.0 and if enabled explicitly), and if not, then fall back to Activity.Current?

Reply:
Probably no as this will be faster?

Reply:
mm. I suppose this is fine then.

We can remove this section as well.
https://github.com/open-telemetry/opentelemetry-dotnet/tree/master/docs/logs/correlation#correlate-logs-with-traces

(That makes me wonder - what is the use case of Ilogger providing this feature in 5.0, as every provider can simply access Activity.Current if they chose to, right?)

Reply:
I think it'll benefit existing providers which already support scope.